library/src/main/java/io/karn/notify/entities/Payload.kt

```diff
@@ -167,6 +167,19 @@ sealed class Payload {
             var showTimestamp: Boolean = true
     )
 
+    /**
+     * Contains configuration that is specific to the progress of a notification.
+     */
+    class Progress constructor(
```

Comment:
Awesome! I'd recommend adding some documentation to these three arguments here. In particular, I'd add some details about how you have to set `showProgress` to `true` to enable the progress mode of the notification and if someone wanted to then make it indeterminate they should also enable the `showIndeterminate` flag and set the progress using the `determinateProgress`.

Reply:
ready